integration/spark2/src/main/scala/org/apache/spark/sql/CarbonCatalystOperators.scala

```diff
@@ -179,6 +179,9 @@ object CountStarPlan {
     if (groupingExpressions.nonEmpty) {
       return false
     }
+    if (partialComputation.isEmpty) {
```

Comment:
Please add  UT if possible.

Reply:
@manishnalla1994 is it possible to get which type of query leads to this issue and add test for it.